src/rest/request.ts

```
@@ -204,7 +237,7 @@ export default async function request(
       : await getNewTokenUsingPasswordGrant(options))
 
   if (!accessToken) {
-    throw new Error('Issue getting OAuth2 authentication token.')
+    throw new Error('Unable to get OAuth2 authentication token.')
```

can we return a better error message here? why were we unable to get the access token, invalid credentials, or some other reason?

---

No, not here. We don't know. Could do something like that in the individual OAuth functions, though. But that's something for another PR.